### DISCUSSION TITLE
"GitHub Free includes three collaborators for private repositories"

### DISCUSSION THREAD

**Discussion by LiorBanai** [community/community#22549] [New to GitHub]
<p>I have private repo in my organization. </p>
<p>Reading the new changes I assumed  the above limitation is no longer  but I still get this message and unable to add more outside collaborator or teams</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/liorbanai">@liorbanai</a>,</p>
<p>Sorry for any confusion! It appears this section of our interface may still be delivering outdated messaging, and our internal team is rolling out an update. In the meanwhile, we’d recommend disregarding this message and inviting additional collaborators, as it should still work for you.</p>
<p>If you run into more errors while inviting collaborators, please <a href="https://support.github.com/contact" title="Contact Support" rel="nofollow noopener">reach out</a> to our private support team, and we’ll take a look for you.</p>

**Comment by LiorBanai** [comment]
<p>Hi <a class="mention" href="/wordsoup">@wordsoup</a> ,</p>
<p>I cannot add more than 3 as the “invite teams or people” button is grayed out.</p>

**Comment by mtilson** [comment]
<p>I can confirm the issue existed (only with “outdated messaging”), but it seems it is fixed now. </p>

**Comment by LiorBanai** [comment]
<p>Yeah I cal also confirmed the issue is fixed.</p>
<p>Thanks!</p>

**Comment by mdell-seradex** [comment]
<p>I am glad to hear this is possible.<br>
From this help page, <a href="https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/adding-outside-collaborators-to-repositories-in-your-organization" rel="noopener nofollow ugc">Adding outside collaborators to repositories in your organization - GitHub Docs</a>, it sounds like we need licenses to do that.<br>
It makes no reference to GitHub Free.</p>
